src/components/standalone/NeMultiTextInput.vue

```diff
@@ -124,44 +126,55 @@ onMounted(() => {
       <span v-if="optional" class="ml-2 text-sm font-normal">{{ optionalLabel }}</span>
     </div>
 
-    <div class="space-y-6">
-      <div class="space-y-4">
-        <div v-for="(item, i) in items" :key="i" class="flex items-start gap-2">
-          <NeCombobox
-            :options="keyOptions"
-            :model-value="keys[i]"
-            @update:model-value="(e: any) => updateModelKey(i, e)"
-            :invalid-message="invalidKeyMessages ? invalidKeyMessages[i] : ''"
-            :placeholder="comboboxPlaceholder"
-            v-if="useKeyCombobox"
-            :noResultsLabel="t('ne_combobox.no_results')"
-            :limitedOptionsLabel="t('ne_combobox.limited_options_label')"
-            :noOptionsLabel="t('ne_combobox.no_options_label')"
-            :selected-label="t('ne_combobox.selected')"
-            :user-input-label="t('ne_combobox.user_input_label')"
-          />
-          <NeTextInput
-            :value="useKeyCombobox ? items[i] : items[i]"
-            @input="updateModelValue(i, $event.target.value)"
-            class="grow"
-            :invalid-message="invalidMessages ? invalidMessages[i] : ''"
-            :disabled="disableInputs"
-            :placeholder="textInputPlaceholder"
-          />
-          <NeButton kind="tertiary" size="md" @click="deleteItem(i)">
-            <font-awesome-icon :icon="['fas', 'trash']" class="h-4 w-4 py-1" aria-hidden="true" />
-          </NeButton>
-        </div>
-        <p v-if="generalInvalidMessage" :class="'mt-2 text-sm text-rose-700 dark:text-rose-400'">
-          {{ generalInvalidMessage }}
-        </p>
-        <NeButton size="md" @click="addItem" :disabled="disableAddButton">
-          <template #prefix>
-            <font-awesome-icon :icon="['fas', 'plus']" class="h-4 w-4" aria-hidden="true" />
-          </template>
-          {{ addItemLabel }}
+    <div class="space-y-4">
+      <div v-for="(item, i) in items" :key="i" class="flex items-start gap-2">
+        <NeCombobox
```

Comment:
Let's always pass all i18n props to `NeCombobox`, soon they will be mandatory. You can find an example [here](https://github.com/NethServer/nethsecurity-ui/blob/78f2f6129bd2731d6cd642f7043c697cab696c15/src/components/standalone/dpi/ManageDpiRuleModal.vue#L480-L485)